icu4c/source/common/rbbi.cpp

```diff
@@ -82,6 +82,18 @@ RuleBasedBreakIterator::RuleBasedBreakIterator(RBBIDataHeader* data, UErrorCode
     }
 }
 
+//-------------------------------------------------------------------------------
+//
+//   Constructor   from a UDataMemory handle to precompiled break rules
+//                 stored in an ICU data file.
```

Comment:
Please also mention this is only intend for private internal usage

Reply:
Resolved